<ui version="4.0" >
 <class>test005Class</class>
 <widget class="QWidget" name="test005Class" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>511</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>test005</string>
  </property>
  <widget class="QLabel" name="background" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblLevel" >
   <property name="geometry" >
    <rect>
     <x>325</x>
     <y>214</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>117</red>
         <green>116</green>
         <blue>118</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>117</red>
         <green>116</green>
         <blue>118</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font" >
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text" >
    <string>10</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblReihen" >
   <property name="geometry" >
    <rect>
     <x>325</x>
     <y>264</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>117</red>
         <green>116</green>
         <blue>118</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font" >
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text" >
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblScore" >
   <property name="geometry" >
    <rect>
     <x>325</x>
     <y>314</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>117</red>
         <green>116</green>
         <blue>118</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font" >
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="layoutDirection" >
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text" >
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="menubg" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSpielstarten" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Spiel starten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHighscores" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Highscores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCredits" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Credits</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBeenden" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Beenden</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
